bin/ci-expected-results

```diff
@@ -70,6 +70,16 @@ if [ "$actual_author_non_authorline_title" != "$expected_author_non_authorline_t
     exit 2
 fi
 
+actual_total=$(curl -v "localhost/search?for=rheumatoid%20arthritis%20(RA)&page=1&per-page=10&sort=relevance&order=desc&use-date=default" | jq ".total")
+if [ ! "$actual_total" ]; then
```

Comment:
Is this checking to see if it the value has been set or not? If so, a better approach would be...

`if [ -z "$actual_total" ]; then`

Which both checks that the variable exists and has been set to anything other than empty.

Reply:
Would it be `if [ ! -z "$actual_total" ]; then`?

Reply:
Sorry yes, for the inverse where it isn't set or empty you need the `!`

Reply:
shellcheck prefers `-n` to `! -z`. It's a great little tool: https://www.shellcheck.net

Reply:
I've not come across ShellCheck before, what a great tool. Bookmark and installed.